Gemfile

```diff
@@ -1,7 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.7.0'
-
-gem 'github-pages'
-gem 'html-proofer', '~> 3.19.4'
-gem 'rubyzip', '~> 2.3.0'
+gem 'webrick', '~> 1.8', '>= 1.8.1'
```

Comment:
Is it really necessary to enforce `>= 1.8.1` as well? `~>1.8` means any version >= 1.8 but smaller 2.0. I guess the latest will be chosen anyway.